src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
```

Comment:
The class 'Rectangle' should not extend 'Figure' if 'Figure' is an abstract class that sets behavior. Consider using an interface to define behavior and use classes to implement them.